src/styles/components/_profile-upload.scss

```diff
@@ -0,0 +1,108 @@
+@import '../abstracts/_abstracts-dir';
+
+.profile-upload {
+  flex-direction: column;
+  box-sizing: border-box;
+
+  // TODO: Remove this when merging to the form
+  background-color: $light;
+
+  @include flex-center;
+
+  &__label {
+    margin-bottom: rem(16px);
+
+    cursor: pointer;
+
+    @include respond-above('lg') {
+      margin-bottom: rem(24px);
+    }
+
+    &--selected {
+      position: relative;
+
+      padding: rem(8px 12px);
+      border: solid 1px $mercury;
+      border-radius: rem(10px);
+
+      @include respond-above('lg') {
+        padding: rem(12px 16px);
+      }
+    }
+  }
+
+  &__input {
+    display: none;
+    visibility: hidden;
+  }
+
+  &__image {
```

Comment:
Try see the size of the default image and after uploaded, it wiggles.

### Before

<img width="263" alt="Screen Shot 2564-03-15 at 15 31 11" src="https://user-images.githubusercontent.com/32289827/111125633-66c13180-85a4-11eb-9169-f265f7fcb2f2.png">

### After

<img width="306" alt="Screen Shot 2564-03-15 at 15 31 46" src="https://user-images.githubusercontent.com/32289827/111125657-6e80d600-85a4-11eb-82c3-cb926599a5b1.png">

Reply:
<img width="303" alt="Screen Shot 2564-03-17 at 14 57 43" src="https://user-images.githubusercontent.com/32289827/111434758-82f2d900-8732-11eb-8e25-c8a020c693f5.png">
<img width="248" alt="Screen Shot 2564-03-17 at 14 57 57" src="https://user-images.githubusercontent.com/32289827/111434765-85553300-8732-11eb-82ba-804b468fbd0e.png">

They still wiggles 🤔